### PR TITLE
add hippyaki

### LINKED ISSUE #586: Akshayan Sinha
Github Profile - hippyaki

### PR DESCRIPTION
**Pull request by hippyaki**
resolve #586

### CODE REVIEW

**Review by deephunt3r — APPROVED** [review]
LGTM 🚀

File: data.json [hunk 1]
```diff
@@ -1091,4 +1091,10 @@
     "imageUrl": "https://user-images.githubusercontent.com/56799372/124683406-90eb8600-deea-11eb-9de0-d4536ed04eeb.png",
     "issueNumber": 578
   }
```
Comment (schmelto) [inline data.json:1093]:
```suggestion
   }, 
```

**Review by schmelto — CHANGES_REQUESTED** [review]

File: data.json (hunk 1 above)
Comment (Panquesito7) [reply data.json:1093]:
```suggestion
  },
```

**Comment by hippyaki** [comment]
Changes made.

**Review by Panquesito7 — APPROVED** [review]
Looks good. Thank you for your contribution! We hope you keep contributing with us! 😉 👍

**Review by schmelto — APPROVED** [review]

**Pull request merged**